src/DataFactory/DataFactoryV2/help/Set-AzDataFactoryV2.md

```diff
@@ -119,12 +345,66 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -ResourceId
+The Azure resource ID.
```

Comment:
@sanjkuma minor: this should say what it's the resource id of (_e.g._, "The Azure resource id of the V2 data factory")

Reply:
Done.